### DISCUSSION TITLE
Pagination is not working for swissup/attributepages extension

### DISCUSSION THREAD

**Discussion by nilaykumardeveloper** [breezefront/community#311] [1. Help]
"swissup/attributepages" version: "1.6.1"
"swissup/breeze-blank" version: "2.5.5"
Magento version: 2.4.6-p3

Steps to Reproduce
Visit: any Swissup attribute pages like https://shop.artuk.org/artist/kyffin-williams-1918-2006
Scroll to the bottom of the page and click page '2'.
The page fails to update to page 2 and just refreshed on the current page.

Kindly check and assist accordingly.

**Comment by vovayatsyuk** [comment]
Please get in touch with Swissuplabs support.

Pagination is not working because https://shop.artuk.org/artist/kyffin-williams-1918-2006?p=3&aln=%E2%9C%93 returns HTML response while Swissup_Ajaxlayerednavigation module expects for JSON response.

p.s. Please ensure that you are using the latest Swissup_Ajaxlayerednavigation and Swisusup_Attributepages modules.

**Comment by nilaykumardeveloper** [comment]
in this case, now closing this discussion.
